src/views/AdminPizza/AdminPizzaExportList.vue

```diff
@@ -0,0 +1,135 @@
+<script setup lang="ts">
+
+import { storeToRefs } from 'pinia';
+import { reactive, ref } from 'vue';
+import Modal from '@/components/Modal.vue';
+import type { Export } from '@/models/timeslot';
+import { usePizzaStore } from '@/stores/pizza.store';
+import { frenchFormatFromDate } from '@/utils';
+
+const pizzaStore = usePizzaStore();
+const { timeslotList, timeslotExportList } = storeToRefs(pizzaStore);
+const {
+  fetchNextTimeslots,
+  fetchAdminDetailTimeslot,
+  fetchTimeslotExports,
+  deleteExport,
+} = pizzaStore;
+
+const timeslotsExpand = reactive<{ [key: number]: boolean }>({});
+const exportToDelete = ref<Export | null>(null);
+
+const closeConfirmDeleteModal = () => {
+  exportToDelete.value = null;
+};
+
+const confirmDeleteExport = async () => {
+  if (exportToDelete.value === null) return;
+
+  await deleteExport(exportToDelete.value.id);
+  closeConfirmDeleteModal();
+};
+
+await fetchNextTimeslots();
+await fetchAdminDetailTimeslot();
+await fetchTimeslotExports();
+
+Object.values(timeslotList.value).forEach((timeslot) => {
+  timeslotsExpand[timeslot.id] = false;
+});
+
+</script>
+
+<template>
+  <h1 class="title">
+    Liste des exports
+  </h1>
+  <div class="mb-6 mt-2 flex flex-col gap-2 px-4">
+    <div
+      v-for="timeslot in timeslotList"
+      :key="timeslot.id"
+      class="flex flex-col overflow-hidden rounded bg-cyan-900"
+    >
+      <div
+        class="flex select-none items-center gap-2 p-2 hover:cursor-pointer hover:bg-cyan-800"
+        @click="timeslotsExpand[timeslot.id] = !timeslotsExpand[timeslot.id]"
+        @keydown.enter="timeslotsExpand[timeslot.id] = !timeslotsExpand[timeslot.id]"
+      >
+        <div class="flex size-8 items-center justify-center text-2xl">
+          <fa-awesome-icon icon="fa-caret-right" :class="{ 'rotate-90': timeslotsExpand[timeslot.id] }"/>
+        </div>
+        <div class="m-0 grow">
+          Créneau {{ frenchFormatFromDate(new Date(timeslot.delivery_time)) }}
+        </div>
+        <div>
+          Export{{ timeslotExportList[timeslot.id].length > 1 ? 's' : '' }}  : {{ timeslotExportList[timeslot.id].length }}
+        </div>
+      </div>
+      <div
+        v-if="timeslotExportList[timeslot.id].length === 0"
+        class="flex items-center justify-center p-3"
+        :class="{ hidden: !timeslotsExpand[timeslot.id] }"
+      >
+        Aucun export n'a été effectué pour ce créneau.
+      </div>
+      <div
+        v-for="timeslotExport in timeslotExportList[timeslot.id]"
+        :key="timeslotExport.id"
+        class="hover:bg-cyan-800"
+        :class="{ hidden: !timeslotsExpand[timeslot.id] }"
+      >
+        <div class="flex select-none items-center gap-2 p-2">
```

Comment:
This works well, but in addition to viewing exports and being able to delete them, I think we could add a download button.
This could be useful if the team need to download again or from another PC the export.
This require a new endpoint tho. What do you think about it ?

Reply:
It's a good idea, but we can do it later and push this in prod for the next mini